plugins/offboard/offboard_impl.cpp

```diff
@@ -160,6 +242,101 @@ void OffboardImpl::set_velocity_body(Offboard::VelocityBodyYawspeed velocity_bod
     send_velocity_body();
 }
 
+void OffboardImpl::send_translation_ned()
+{
+    // by default ingore all setpoints
+    uint16_t IGNORE_X = (1 << 0);
+    uint16_t IGNORE_Y = (1 << 1);
+    uint16_t IGNORE_Z = (1 << 2);
+    uint16_t IGNORE_VX = (1 << 3);
+    uint16_t IGNORE_VY = (1 << 4);
+    uint16_t IGNORE_VZ = (1 << 5);
+    uint16_t IGNORE_AX = (1 << 6);
+    uint16_t IGNORE_AY = (1 << 7);
+    uint16_t IGNORE_AZ = (1 << 8);
+    // uint16_t IS_FORCE = (1 << 9);
+    uint16_t IGNORE_YAW = (1 << 10);
+    uint16_t IGNORE_YAW_RATE = (1 << 11);
+
+    float x = 0.0f;
+    float y = 0.0f;
+    float z = 0.0f;
+    float vx = 0.0f;
+    float vy = 0.0f;
+    float vz = 0.0f;
+    float yaw = 0.0f;
+    float yawspeed = 0.0f;
+    const float afx = 0.0f;
+    const float afy = 0.0f;
+    const float afz = 0.0f;
+
+    _mutex.lock();
+    if (std::isfinite(_translation_ned.yaw_deg)) {
+        yaw = to_rad_from_deg(_translation_ned.yaw_deg);
+        // enable yaw
+        IGNORE_YAW &= ~(1 << 10);
```

Comment:
So this is a bit inverted in how it was planned. The goal was to create readable constants for the ignore flags, so that we don't have to use these magic `<< 10` shifters anywhere else.

However, what you're doing is more bitwise operations that are hard to check down here.

I'd suggest to do something like that:
```
const uint16_t IGNORE_YAW = (1 << 10);
...

uint16_t ignore_flags = 0;

if (std::isfinite(_translation_ned.yaw_deg)) {
    yaw = to_rad_from_deg(_translation_ned.yaw_deg);
} else {
    ignore_flags |= IGNORE_YAW;
}
```